ex2-shai/bootstrap.sh

```diff
@@ -4,12 +4,11 @@ which git > /dev/null 2>&1
 if [[ $? -eq 1 ]]; then        
         yum install http://opensource.wandisco.com/centos/7/git/x86_64/wandisco-git-release-7-2.noarch.rpm
         yum install git
-        git config --global user.name <Full Name>
-        git config --global user.email <email address>
+        git config --global user.name Shai Kamenker
```

Comment:
It's a good habit not to hard-coded values.
I suggest to parameterized.

Reply:
+1